core/modules/wishlist/components/WishlistButton.ts

```diff
@@ -3,5 +3,10 @@ export const WishlistButton = {
     toggleWishlist () {
       this.$store.dispatch('ui/toggleWishlist')
     }
+  },
+  computed: {
+    wishlistItemsCount () {
```

Comment:
you could use `...mapGetters` for that to not duplicate logic and then use `getWishlistItemsCount`
```js
...mapGetters('wishlist', ['getWishlistItemsCount'])
```